manifests/config.pp

```diff
@@ -30,8 +28,8 @@
   }
 
   # Custom jails definition
-  if $config_custom_jails {
-    create_resources('fail2ban::jail', $config_custom_jails)
+  unless $::fail2ban::custom_jails.empty {
```

Comment:
I thought `create_resources` handles empty hashes as well which would make this check redundant.